scripts/packages/DebugAdapter/Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "DebugAdapter"
+uuid = "b9dc54e9-555d-4545-8d72-fb33852127ad"
+version = "0.1.0"
+
+[deps]
+JuliaInterpreter = "aa1ae85d-cabe-5617-a682-6adf51b2e16a"
```

Comment:
This will load the "real" version of JuliaInterpreter.jl (i.e. the one registered into General), no ? 
I guess you need to checkin Manifest.toml for this package ?

Reply:
Actually, I think I didn't have to add this file at all because it will never be used in our setting: DebugAdapter is only being loaded via `include` in our extension, never via the normal package load mechanism. The one that really matters in the `Project.toml` for `VSCodeDebugger`. My next PR is actually moving DebugAdapter into a proper external package in any case, so this file will have a short lifetime no matter what ;)

Reply:
okay, let's focus on the PR.